cypress/integration/test_create_sales_invoice.js

```diff
@@ -0,0 +1,69 @@
+
+context('Sales Invoice Creation', () => {
+	before(() => {
+		cy.login();
+	});
+	
+	it('Create Item', () => {
```

Comment:
This item + customer creation are not tests but fixtures required for upcoming tests, these should be part of `before`.